### DISCUSSION TITLE
SD.Next 2024-06-23 - UI modern - how to send image to img2img ?

### DISCUSSION THREAD

**Discussion by cgidesign-de** [vladmandic/sdnext#3288] [Q&A] [answered]
New modern UI looks nice - thanks for making it.
Trying to understand it at the moment.

In old UI there was "send to img2img" for an image created in "text2img".
I am missing this in the modern UI.

Has this "img2img" concept changed ?

**Comment by brknsoul** [comment]
![image](https://github.com/vladmandic/automatic/assets/1969381/7f3853e5-e1bf-4ac7-9948-cf6d98939161)

**Reply by cgidesign-de** [reply]
Hm, seems so your UI is different than mine.
This is what I see after a fresh install of latest SD.Next and a run of text2img (Firefox on Windows).

![UI-Modern](https://github.com/vladmandic/automatic/assets/115958660/9a3ad380-efba-404b-ba6f-0692cc7af792)

**Reply by brknsoul** [reply, marked as answer]
There's an option somewhere in Settings to re-enable the legacy Text and Image tabs.

![image](https://github.com/vladmandic/automatic/assets/1969381/9fbced1c-0dcc-4248-baf0-f0da26f9a2ff)


You can do all the old Text and Image functions in the Control tab, but I'm not well versed in how to this.

**Reply by cgidesign-de** [reply]
Ah, that was it. Now I have the buttons like in your screenshot. Thank you.

Is there a reason why the default has changed to not show those?
Are we expected to do img2img in another way than in the old UI?

You mentioned "Control tab". I admit, for me it is yet a mistery how to "recreate" img2img there.

**Reply by brknsoul** [reply]
You'd enable Input Image check box. drag an image in to the Input section, scroll down a bit and click Inpaint;
![image](https://github.com/vladmandic/automatic/assets/1969381/f37aee31-c0a3-49c2-944e-28116e405831)

But there's a few issues with the modernui, like lack of zooming in and out or panning.

**Reply by cgidesign-de** [reply]
Ok, I'll check that out.
